### PR TITLE
[DOC] update the tutorial todo app

### PR DESCRIPTION
**Pull request by ged-odoo**

### CODE REVIEW

File: doc/learning/quick_start.md [hunk 1]
```diff
@@ -45,30 +47,24 @@ Now, `index.html` should contain the following:
   <head>
     <title>Hello Owl</title>
     <script src="owl.js"></script>
-    <script src="app.js"></script>
   </head>
-  <body></body>
+  <body>
+    <script src="app.js"></script>
```
Comment (Polymorphe57) [inline doc/learning/quick_start.md:52]: root.js?

File: doc/learning/quick_start.md [hunk 2]
```diff
@@ -45,30 +47,24 @@ Now, `index.html` should contain the following:
...
   static template = xml`<div>Hello Owl</div>`;
 }
 
-// Setup code
-function setup() {
-  mount(App, target: { document.body })
-}
-
-whenReady(setup);
+mount(Root, document.body);
```
Comment (Polymorphe57) [inline doc/learning/quick_start.md:67]: no more whenReady?

Comment (Polymorphe57) [reply doc/learning/quick_start.md:67]: linked to the new location of script tag, I guess

File: doc/learning/quick_start.md [hunk 3]
```diff
@@ -110,37 +108,33 @@ Now, `index.html` should contain the following:
...
-  mount(App, { target: document.body });
-}
+import { Root } from "./root.js";
 
-owl.utils.whenReady(setup);
+mount(Root, document.body);
 ```
 
-The `main.js` file import the `app.js` file. Note that the import statement has
+The `main.js` file import the `root.js` file. Note that the import statement has
```
Comment (sdegueldre) [inline doc/learning/quick_start.md:137]: import => imports

File: doc/learning/tutorial_todoapp.md [hunk 4]
```diff
@@ -517,219 +478,228 @@ First, let us update the `Task` template, css and js:
...
-deleteTask(ev) {
-    const index = this.tasks.findIndex(t => t.id === ev.detail.id);
+deleteTask(task) {
+    const index = this.tasks.findIndex(t => t.id === task.id);
     this.tasks.splice(index, 1);
 }
 ```
 
+Notice that the `onDelete` prop is defined with a `.bind` suffix: this is a special
+suffix that make sure the function callback is bound to the component.
```
Comment (sdegueldre) [inline doc/learning/tutorial_todoapp.md:503]: make => makes

**Review by sdegueldre — APPROVED** [review]

**Pull request merged**